js/packages/components/dropdowns/Dropdown.stories.tsx

```diff
@@ -0,0 +1,104 @@
+import { storiesOf } from '@storybook/react-native'
+import Long from 'long'
+import React, { useState } from 'react'
+import { ScrollView, Text, View } from 'react-native'
+
+import beapi from '@berty/api'
+
+import { TextualDropdown, MembersDropdown } from '.'
+import { AppDecorator } from '../../../.storybook/preview'
+
+const Spacer = () => <View style={{ height: 16 }} />
```

Comment:
Same here, can you add styles to a `Stylesheet` variable at the bottom of the file ?